docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -106,6 +106,31 @@ i.e. you cannot bind a Horizontal Pod Autoscaler to a replication controller and
 The reason this doesn't work is that when rolling update creates a new replication controller,
 the Horizontal Pod Autoscaler will not be bound to the new replication controller.
 
+## Support for cooldown/delay
+
+When managing the scale of a group of replicas using the Horizontal Pod Autoscaler,
+it is possible that the number of replicas keeps fluctuating frequently due to
+dynamic nature of the metrics evaluated. This is sometimes referred to as "thrashing".
```

Comment:
add "the": "...`the` dynamic nature of the metrics evaluated."

use italics instead of quotes: "This is sometimes referred to as *`thrashing`*."

Reply:
Done